test/test_sign_command.rb

```diff
@@ -36,6 +36,24 @@ def test_sign
     assert_equal [], output
   end
 
+  def test_static_sign
+    @cmd.options[:args] = [@gem_path]
+    @cmd.options[:identity_token] = access_token
```

Comment:
Why `access_token` here? That seems different from the code, which is using `token`, and from the CLI surface, which uses `identity_token`.